CHANGELOG.md

```diff
@@ -51,6 +51,9 @@ generate in an area where it previously did not (resulting in a chunk border).
 
 ### Fixed
 
+## [v1.3.4]
```

Comment:
Section titles and their anchors for new releases are automated by CI so leave things under the 'Unreleased' section

Reply:
oh i see

Reply:
I should probably leave a comment there explaining this tho